tests/files/c_source/vehicle.h

```diff
@@ -256,6 +256,56 @@
 #define VEHICLE_FRAME_ID_RT_SB_ACCEL (0x94a6c22U)
 #define VEHICLE_FRAME_ID_RT_SB_GYRO_RATES (0x94a6d22U)
 
+#define VEHICLE_RT_SB_INS_STATUS_INS_STATUS_VAL_CONVERGED (2U)
+#define VEHICLE_RT_SB_INS_STATUS_INS_STATUS_VAL_INITIALISED (1U)
+#define VEHICLE_RT_SB_INS_STATUS_INS_STATUS_VAL_UNINITIALISED (0U)
+
+#define VEHICLE_RT_SB_INS_ATTITUDE_VALIDITY_YAW_VAL_VALID (1U)
+#define VEHICLE_RT_SB_INS_ATTITUDE_VALIDITY_YAW_VAL_INVALID (0U)
+#define VEHICLE_RT_SB_INS_ATTITUDE_VALIDITY_PITCH_VAL_VALID (1U)
+#define VEHICLE_RT_SB_INS_ATTITUDE_VALIDITY_PITCH_VAL_INVALID (0U)
+#define VEHICLE_RT_SB_INS_ATTITUDE_VALIDITY_ROLL_VAL_VALID (1U)
+#define VEHICLE_RT_SB_INS_ATTITUDE_VALIDITY_ROLL_VAL_INVALID (0U)
```

Comment:
Lets separate the signals with a blank line as it makes it easier to read. I think it adds more value than blank line between messages.

``` c
#define VEHICLE_RT_SB_INS_ATTITUDE_VALIDITY_YAW_VAL_VALID (1U)
#define VEHICLE_RT_SB_INS_ATTITUDE_VALIDITY_YAW_VAL_INVALID (0U)

#define VEHICLE_RT_SB_INS_ATTITUDE_VALIDITY_PITCH_VAL_VALID (1U)
#define VEHICLE_RT_SB_INS_ATTITUDE_VALIDITY_PITCH_VAL_INVALID (0U)

#define VEHICLE_RT_SB_INS_ATTITUDE_VALIDITY_ROLL_VAL_VALID (1U)
#define VEHICLE_RT_SB_INS_ATTITUDE_VALIDITY_ROLL_VAL_INVALID (0U)
```

Reply:
Done